openslides_backend/migrations/migrations/0062_unset_presence_of_removed_users.py

```diff
@@ -0,0 +1,63 @@
+from collections import defaultdict
+
+from datastore.migrations import BaseModelMigration
+from datastore.writer.core import BaseRequestEvent, RequestUpdateEvent
+
+from openslides_backend.shared.patterns import fqid_from_collection_and_id
+
+
+class Migration(BaseModelMigration):
+    """
+    This migration adds the user_id "-1" to all existing action_workers.
+    This is the number usually used for calls using the internal route.
+    """
+
+    target_migration_index = 63
+
+    def migrate_models(self) -> list[BaseRequestEvent] | None:
+        present_users_per_meeting: dict[int, list[int]] = defaultdict(list)
+        meetings_per_present_user: dict[int, list[int]] = defaultdict(list)
+        meetings = self.reader.get_all(
+            "meeting", ["group_ids", "present_user_ids", "meeting_user_ids"]
+        )
+        users = self.reader.get_all(
+            "user", ["is_present_in_meeting_ids", "meeting_user_ids"]
+        )
+
+        def helper() -> None:
+            if not (
+                (meeting_user_ids := user.get("meeting_user_ids", []))
```

Comment:
Where does helper get his user from?

Reply:
Added arguments. Before the nested function would use all local variables in the enclosing scope.

Reply:
Due to refactoring this function does not exist anymore.